client/payment-methods/test/index.js

```diff
@@ -143,4 +173,31 @@ describe( 'PaymentMethods', () => {
 			'sofort',
 		] );
 	} );
+
+	test( 'renders the Feedback/Disable Header when UPE is enabled', () => {
+		render(
+			<WcPayUpeContextProvider defaultIsUpeEnabled={ true }>
+				<PaymentMethods />
+			</WcPayUpeContextProvider>
+		);
+		const disableUPEButton = screen.getByRole( 'button', {
```

Comment:
(minor) when using the `getBy*` query, if the element is not present in the DOM RTL will throw an exception, making the test fail.
So if the element is not present, it is `getBy*` that will make the test fail, not the `expect`
```suggestion
		const disableUPEButton = screen.queryByRole( 'button', {
```

Reply:
For sake of consistency I'll revert it back to `queryByRole`; I like that getByRole errors automatically but having each test with an assertion is probably more readable.